src/protocol/messagedefs.jl

```diff
@@ -24,6 +24,7 @@ const workspace_applyEdit_request_type = JSONRPC.RequestType("workspace/applyEdi
 const workspace_configuration_request_type = JSONRPC.RequestType("workspace/configuration", ConfigurationParams, Vector{Any})
 
 const julia_getModuleAt_request_type = JSONRPC.RequestType("julia/getModuleAt", TextDocumentPositionParams, String)
+const julia_getDocAt_request_type = JSONRPC.RequestType("julia/getDocAt", TextDocumentPositionParams, String)
```

Comment:
Should we use the struct here that has the version information included that @pfitzseb just added to the code base? We're potentially going to run into the same issue here, right?

Reply:
Probably a good idea, yes.

Reply:
What should we return as `JSONRPCError`'s `data` field ? `getModuleAt` and `getBlockAt` return `nothing`.

Reply:
I think `nothing` is fine, not sure what one is supposed to do with that field ;)

Reply:
hehe, I get it :)